dolphin/db/sqlalchemy/api.py

```diff
@@ -173,7 +173,12 @@ def access_info_create(context, values):
 
 def access_info_update(context, access_info_id, values):
     """Update a storage access information with the values dictionary."""
-    return NotImplemented
+    session = get_session()
+    with session.begin():
+        result = session.query(AccessInfo) \
+            .filter(AccessInfo.storage_id == access_info_id) \
+            .first().update(values)
+    return result
```

Comment:
should check if result is None, should report Exception or do like this:
```
with session.begin():
    result = _access_info_get(context, access_info_id, session).update(values)
    return result
```

Reply:
done